cachito/workers/pkg_managers/npm.py

```diff
@@ -25,7 +25,7 @@
 log = logging.getLogger(__name__)
 
 
-def _get_deps(package_lock_deps, file_deps_allowlist, _name_to_deps=None):
+def _get_deps(package_lock_deps, file_deps_allowlist, package_lock_lockfileversion, workspaces, _name_to_deps=None):
```

Comment:
Just be careful that this function is also clalled in line 121, and adding these new mandatory args is breaking the code when it reaches there.

Reply:
Sure, I just wanted your opinions first :)